website/static/schemas/next/bridging/findIntentsForContextRequest.schema.json

```
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/findIntentsForContextRequest.schema.json",
+  "title": "FindIntentsForContextRequest",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "bridgeRequest.schema.json"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "type": {
+          "const": "findIntentsForContextRequest"
+        },
+        "payload": {
+          "type": "object",
+          "properties": {
+            "context": {
+              "$ref": "../context/context.schema.json"
+            }
+          },
+          "required": ["context"],
+          "additionalProperties": false
+        },
+        "meta": {
+          "properties": {
+            "requestGuid": true,
+            "timestamp": true,
+            "destination": true,
+            "source": {
+              "$ref": "../api/appIdentifier.schema.json"
+            }
```

I made a mistake on this one, it could come from the intent resolver, which (i suppose) is the desktop agent
```suggestion
            "source": true
```